mobile/Screens/PocketPromptScreen.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import {
+  StyleSheet, Text, View, TouchableOpacity, Dimensions,
+} from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import axios from 'axios';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#151716',
```

Comment:
I think the Figma has a slightly different background now if you can replace this Hex code with a png!